doc/releases/release-notes-3.7.rst

```diff
@@ -379,6 +379,9 @@ Boards & SoC Support
   * nRF54H20 PDK (pre-release) converted to :ref:`nrf54h20dk_nrf54h20`
   * PPR core target in :ref:`nrf54h20dk_nrf54h20` runs from RAM by default. A
     new ``xip`` variant has been introduced which runs from MRAM (XIP).
+  * Refactored :ref:`beagleconnect_freedom` external antenna switch handling.
+  * Fix zephyr:code-sample:`pwm-blinky` code sample to demonstrate PWM support for
+    :ref:`beagleconnect_freedom`.
```

Comment:
I think the following also need to be added:
- gpio-i2c-switch: https://github.com/zephyrproject-rtos/zephyr/pull/64693
- subg: https://github.com/zephyrproject-rtos/zephyr/pull/65048

Not sure if the 2nd one is part of beagle notes though

Reply:
Do not see those in `git log v3.6.0..main`. Perhaps they went into 3.6?